ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -41,6 +41,47 @@ EOF
     fi
 }
 
+function update_cloud_credentials(){
+    platform=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}')
+    cco_mode=$(oc get cloudcredential cluster -o jsonpath='{.spec.credentialsMode}')
+    if (( SOURCE_MINOR_VERSION == TARGET_MINOR_VERSION )) || (( SOURCE_MINOR_VERSION < 13 )) || [[ "${cco_mode}" != "Manual" ]]; then
+        echo "No need update cloud credentials" && return
+    fi
+    preCredsDir="/tmp/pre-include-creds"
+    tobeCredsDir="/tmp/tobe-include-creds"
+    mkdir "${preCredsDir}" "${tobeCredsDir}"
+    # Extract all CRs from live cluster with --included
+    if ! oc adm release extract --to "${preCredsDir}" --included --credentials-requests; then
+        echo "Failed to extract CRs from live cluster!" && exit 1
+    fi
+    if ! oc adm release extract --to "${tobeCredsDir}" --included --credentials-requests "${TARGET}"; then
```

Comment:
The `--include` would help only extract necessary credential CR manifests against `TARGET` even the current  cluster is not using the TARGET payload?

Reply:
yes